systemd/SMGC@.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Garbage Collector for SR %I
+DefaultDependencies=no
+RequiresMountsFor=/run/sr-mount/%I
```

Comment:
Not true as this will run on LVM SRs as well which do not have a mount point. It needs to work without a `RequiresMountsFor` settng

Reply:
Ah yes. Cut&paste error. Had no actual effect on a direct "start" command which would explain why the tests all worked.